torch/legacy/nn/MarginRankingCriterion.py

```diff
@@ -60,7 +60,7 @@ def updateGradInput(self, input, y):
             self.mask = self.mask.resize_as_(input[0]).copy_(dist)
             mask = self.mask
 
-            torch.ge(dist, 0, out=mask)
+            torch.ge_t(dist, 0, out=mask)
```

Comment:
I wouldn't say it's minor, as it's our user interface. It's only irrelevant to this PR, and I think reverting it is the best solution.